addons/l10n_it_edi/models/res_partner.py

```diff
@@ -31,6 +30,13 @@ class ResPartner(models.Model):
             "Destination Code must have between 6 and 7 characters."),
     ]
 
+    def _is_pa(self):
+        """
+            Returns True if the destination of the FatturaPA belongs to the Public Administration.
+        """
```

Comment:
You are not consistent with any of your docstrings... it makes me crazy. Choose a way and stick to it. The most common one is:
```py
""" blablabla

:param ...
:return ...
"""
```

Without any parameter:
```suggestion
        """ blablabla """
```